.github/workflows/ci.yml

```
@@ -197,11 +179,39 @@ jobs:
+    steps:
+    - name: Get OMDev version hash
+      run: |
+        cmake --version
+        git ls-remote https://openmodelica.org/git/OMDev.git HEAD > omdev_hash.txt
+        echo "cache-key=OMDev-${{ hashFiles('omdev_hash.txt') }}" >> $GITHUB_OUTPUT
+    - name: Restore the OMDev cache
+      uses: actions/cache/restore@v4
+      id: cache-omdev
+      with:
+        path: C:/OMDev
+        key: ${{ steps.cache-omdev.outputs.cache-key }}
```

actionlint (1.7.8)

property "cache-omdev" is not defined in object type {}

(expression)